configuration/constants.lua

```diff
@@ -0,0 +1,9 @@
+local constants = {
+    machineAddPort = 0xADD,
```

Comment:
Is this an actual port number reserved in oc or are we just using this hex because it's easy to remember?

Reply:
Easy to remember